api/protobuf/api.proto

```diff
@@ -173,7 +171,7 @@ message GetSealResponse {
 }
 
 message ListSealsForHeightRequest {
-  uint64 height = 1 [(tagger.tags) = "validate:\"required\"" ];
+  uint64 height = 1;
```

Comment:
I assume these are the validation tags Alexey was referring in the thread. Are they required since we remove `tagger/tagger.proto` in this PR?

Reply:
added it back, made it a dep on `buf`